contrib/python/pypodman/pypodman/lib/config.py

```diff
@@ -125,8 +157,8 @@ def resolve_configuration(self, args):
             if dir_ is None:
                 continue
             with suppress(OSError):
-                with open(os.path.join(dir_, 'pypodman/pypodman.conf'),
-                          'r') as stream:
+                with open(os.path.join(dir_,
+                                       'pypodman/pypodman.conf')) as stream:
```

Comment:
Just double checking to verify that you meant to remove the 'r' here....

Reply:
@TomSweeneyRedHat Yup read is default.